tests/integration/select-component-test.js

```diff
@@ -672,7 +673,62 @@ test('Dropdown does not open on key event when select is disabled', function(ass
   selectElement.focus();
   pressDownArrow(selectElement);
 
-  andThen(() => {
+  return andThen(() => {
     assert.ok(isNotPresent('.ember-select-results', selectElement), 'The dropdown does not open');
   });
 });
+
+test('Collapsed group can be expanded, collapsed', function(assert) {
+  function data() {
+    return [
+      {name: 'Sparrow', sound: 'Squawk'},
+      {name: 'Crow', sound: 'Squawk'},
+      {name: 'Dog', sound: 'bark'},
+      {name: 'Wolf', sound: 'Bark'},
+      {name: 'Sea Lion', sound: 'Bark'}
+    ];
+  }
+  function renderedItems(selectElement) {
```

Comment:
How different is it from `getOptionsInSelect` in `tests/helpers/components/select-helpers.js`?

https://github.com/Addepar/Iverson/blob/5382b4ea68583a521838efff83fef8d8ce26cb91/iverson/tests/helpers/components/select-helpers.js#L288-L296

Reply:
😉 different codebase. But it looks like the same thing, yes.

Reply:
lol, sorry